docs/specs/schema.yml

```diff
@@ -65,40 +65,39 @@ outputs:
 # Support schema documents with Href content
 inputs:
   docfx.yml:
-  # TODO: why is there no output for this toc file?
-  TOC.yml:
+  docs/A/TOC.yml:
   docs/TOC.yml:
   docs/a.yml: |
     #YamlMime:ContextObject
     brand: azure
     breadcrumb_path: TOC.yml
-    toc_rel: ../TOC.yml
+    toc_rel: A/TOC.yml
```

Comment:
what's that schema defination?

Reply:
https://github.com/dotnet/docfx/blob/v3/schemas/ContextObject.json#L25